client/app/react-components/DynamicTable.js

```diff
@@ -0,0 +1,347 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pagination } from 'react-bootstrap';
+import { extend, findIndex, filter, isFunction, isString, isUndefined, trim } from 'lodash';
+import $ from 'jquery';
```

Comment:
Is jQuery used here? I see the `$` being used, but those are [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).